server/db/models/song.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var _ = require('lodash');
+var Sequelize = require('sequelize');
+
+module.exports = function (db) {
+
+     db.define('songs', {
+        title: {
+            type: Sequelize.STRING,
+            allowNull: false
+        },
+        description: {
+            type: Sequelize.TEXT,
+            allowNull: false
+        },
+        yearComposed: {
+            type: Sequelize.INTEGER
+        },
+        fileName: {
+            type: Sequelize.STRING,
+            allowNull: false
+        },
+        price: {
+            type: Sequelize.DECIMAL,
+            allowNull: false
+        },
+        inventoryQuantity: {
+            type: Sequelize.INTEGER,
+            allowNull: false
+        },
+        imageURL: {
+            type: Sequelize.STRING,
+            allowNull: false
```

Comment:
i was thinking that the default thumbnail was going to be the imageUrl. Thats what i had in mind when i suggested imageUrl